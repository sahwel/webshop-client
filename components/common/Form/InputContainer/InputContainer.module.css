@tailwind components;

@layer components {
    .input_container{
        @apply grid gap-5
    }

}
    
