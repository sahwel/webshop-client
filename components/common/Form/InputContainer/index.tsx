import { FunctionComponent, ReactNode } from 'react';
import styles from './InputContainer.module.css';

interface InputContainerProps {
  children: ReactNode;
}

const index: FunctionComponent<InputContainerProps> = ({ children }) => {
  return <div className={styles.input_container}>{children}</div>;
};

export default index;
