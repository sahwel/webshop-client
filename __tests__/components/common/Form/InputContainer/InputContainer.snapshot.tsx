import InputContainer from '@/components/common/Form/InputContainer';
import { render } from '@testing-library/react';

describe('InputContainer snapshot', () => {
  it('renders InputContainer unchanged', () => {
    const { container } = render(
      <InputContainer children={<p>test children</p>} />
    );
    expect(container).toMatchSnapshot();
  });
});
