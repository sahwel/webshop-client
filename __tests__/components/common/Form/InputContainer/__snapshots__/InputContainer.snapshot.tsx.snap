// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputContainer snapshot renders InputContainer unchanged 1`] = `
<div>
  <div
    class="input_container"
  >
    <p>
      test children
    </p>
  </div>
</div>
`;
